index.html

```diff
@@ -196,10 +196,26 @@ <h3 class="connect">LET'S CONNECT</h3>
               </li>
             </ul>
           </li>
-          <li class="cat-list-items">Framwork <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg" alt=""></span></li>
-          <li class="cat-list-items">Skills <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg" alt=""></span></li>
+          <li class="cat-list-items">Framwork <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg"
+                alt=""></span></li>
+          <li class="cat-list-items">Skills <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg"
+                alt=""></span></li>
         </ul>
       </section>
+      <section class="contact-section">
+        <form action="" method="get" class="contact-form">
+          <h2 class="contacts-heading">Contact Me</h2>
+          <p class="message">
+            If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I’d love to help with it
+          </p>
+          <input type="text" name="full-name" id="name" placeholder="name" class="contacts-input">
+          <input type="text" name="email" id="email" placeholder="email address" class="contacts-input">
```

Comment:
- [ ] Kindly note that the `type` attribute of your email input should  be `email` and not `text` to ensure that the correct data is entered